occ_status.cpp

```diff
@@ -566,6 +519,38 @@ void Status::occReadStateNow()
     }
 }
 
+void Status::setSensorValueToNaN() const
+{
+    for (const auto& [sensorPath, occId] : existingSensors)
+    {
+        if (occId == instance)
```

Comment:
Isn't the sensors stored in the status object (meaning you don't need to check the instance ID)?
They used to all be in the manager, but I assume they could move to status object (since I dont think any are shared between occs)